.github/workflows/serverless.yaml

```diff
@@ -0,0 +1,35 @@
+name: Serverless
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install scylla-ccm
+        run: pip3 install https://github.com/scylladb/scylla-ccm/archive/master.zip
+
+      - name: Create serverless cluster
+        run: |
+          ccm create serverless -i 127.0.1. -n 1 --scylla -v release:5.1.6
```

Comment:
Can ccm pick the latest released version instead of choosing a particular one? Otherwise we will have to remember to update the version from time to time.

Reply:
Understood, thanks.